packages/terra-modal/src/Modal.jsx

```diff
@@ -97,13 +121,14 @@ class Modal extends React.Component {
           onOpen,
           onUpdate,
           role,
+          onRequestClose,
            ...customProps } = this.props;
 
     return (
       <Portal
         isOpened={isOpened}
-        closeOnEsc={closeOnEsc}
-        closeOnOutsideClick={closeOnOutsideClick}
+        closeOnEsc={false}
```

Comment:
https://github.com/cerner/terra-core/pull/321/commits/de085f27a1f398ca35f224f1e84353e61cb69c4d
Remove the props set to false. We get the same behavior without them set.